projboard/models.py

```diff
@@ -52,6 +52,62 @@ def get_user_by_nickname(nickname):
             return None
         return user
 
+    @staticmethod
+    def create_user(email, password, name, nickname):
+        """
+         Method creates a new user
+        :param email: user's email
+        :param password: user's password
+        :param name: user's name
+        :param nickname: user's nickname
+        :return: new user
+        """
+        user = User.objects.create_user(email=email, password=password, name=name, nickname=nickname)
```

Comment:
Should be:
`user = User(email=email,...)`
And not:
`User.objects.create_user(email=email, password=password, name=name, nickname=nickname)`

Reply:
I seen the last commit, ok for me.